test/evergreen/compatibility_test_for_releases.sh

```diff
@@ -149,13 +150,28 @@ upgrade_downgrade()
         done
 }
 
+
+if [ $# -eq 1 ]; then
+    if [ $1 = "-l" ]; then
+        echo "=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-="
+        echo "Performing long compatibility test run"
+        echo "=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-="
+    else
+        echo "Usage: \tcompatibility_test_for_releases [-l]"
+        echo "\t-l\trun additional variants of wiredtiger"
+        exit 1
+    fi
+fi
+
 # Create a directory in which to do the work.
 top="test-compatibility-run"
 rm -rf "$top" && mkdir "$top"
 cd "$top"
 
 # Build the branches.
-(build_branch mongodb-3.4)
+if [ $# -eq 1 ]; then
```

Comment:
Instead of checking `if [ $# -eq 1 ]` here and later in many places, could you set a variable after line 155 and check the variable?